controllers/cloud.redhat.com/version.txt

```diff
@@ -1 +1 @@
-master
\ No newline at end of file
+v0.46.0-149-gea2bc060
```

Comment:
it feels like this shouldn't be updated, but I'm not 100% sure.

Reply:
This accurate

Reply:
I think you're right. It gets updated when you do a clowder install so I probably just shouldn't have committed it.